applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.h

```diff
@@ -0,0 +1,319 @@
+ * The element is designed to use an explicit integration method.
+ * @author Riccardo Tosi
+ */
+template< unsigned int TDim, unsigned int TNumNodes>
+class SymbolicDConvectionDiffusionExplicit : public SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+        typedef SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes> BaseType;
+        typedef typename BaseType::ElementData ElementData;
+        typedef Node < 3 > NodeType;
+        typedef Geometry<NodeType> GeometryType;
+        typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
+        typedef Vector VectorType;
+        typedef Matrix MatrixType;
+        typedef std::size_t IndexType;
+        typedef std::vector<std::size_t> EquationIdVectorType;
+        typedef std::vector< Dof<double>::Pointer > DofsVectorType;
+        typedef GeometryData::IntegrationMethod IntegrationMethod;
+
+    /// Pointer definition of SymbolicDConvectionDiffusionExplicit
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(SymbolicDConvectionDiffusionExplicit);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    //Constructors.
+    SymbolicDConvectionDiffusionExplicit(
+        IndexType NewId,
+        GeometryType::Pointer pGeometry);
+    SymbolicDConvectionDiffusionExplicit(
+        IndexType NewId,
+        GeometryType::Pointer pGeometry,
+        Properties::Pointer pProperties);
+
+    /// Default constuctor.
+
+    /// Destructor.
+    virtual ~SymbolicDConvectionDiffusionExplicit();
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    Element::Pointer Create(
+        IndexType NewId,
+        NodesArrayType const& ThisNodes,
+        Properties::Pointer pProperties) const override;
+
+    Element::Pointer Create(
+        IndexType NewId,
+        GeometryType::Pointer pGeom,
+        Properties::Pointer pProperties) const override;
+
+    void CalculateLocalSystem(
+        MatrixType& rLeftHandSideMatrix,
+        VectorType& rRightHandSideVector,
+        const ProcessInfo& rCurrentProcessInfo) override
+    {
+        KRATOS_TRY;
+        KRATOS_ERROR << "Calling the CalculateLocalSystem() method for the explicit Convection-Diffusion element.";
+        KRATOS_CATCH("");
+    }
+
+    void CalculateRightHandSide(
+        VectorType& rRightHandSideVector,
+        const ProcessInfo& rCurrentProcessInfo) override
+    {
+        KRATOS_TRY;
+        KRATOS_ERROR << "Calling the CalculateRightHandSide() method for the explicit Convection-Diffusion element. Call the CalculateRightHandSideInternal() instead.";
+        KRATOS_CATCH("");
+    }
+
+    void AddExplicitContribution(
+        const ProcessInfo &rCurrentProcessInfo) override;
+
+    void Initialize(
+        const ProcessInfo& rCurrentProcessInfo) override;
+
+    void FinalizeSolutionStep(
+        const ProcessInfo& rCurrentProcessInfo) override;
```

Comment:
```suggestion
    void AddExplicitContribution(const ProcessInfo &rCurrentProcessInfo) override;

    void Initialize(const ProcessInfo& rCurrentProcessInfo) override;

    void FinalizeSolutionStep(const ProcessInfo& rCurrentProcessInfo) override;
```
Single line for single argument functions

Reply:
Ok